tests/layer_tests/tensorflow2_keras_tests/test_tf2_keras_stackedrnncells.py

```diff
@@ -31,18 +31,16 @@ def create_keras_stackedrnncells_net(self, input_names, input_shapes, input_type
         return tf2_net, ref_net
 
```

Comment:
Changes affect the tests: 

```
=========================== short test summary info ============================
ERROR install/tests/layer_tests/tensorflow2_keras_tests/test_tf2_keras_upsampling2d.py::TestKerasUpSampling2D - Failed: tensorflow2_keras_tests/test_tf2_keras_upsampling2d.py::TestKerasUpSampling2D::test_keras_upsampling2d_bilinear: in "parametrize" the number of names (6):
  ['input_names', 'input_shapes', 'input_type', 'size', 'data_format', 'interpolation']
must be equal to the number of values (1):
  ((['x1'], [[1, 6, 2, 1]], tf.float32, (3, 1), 'channels_last', 'bilinear'),)
ERROR install/tests/layer_tests/tensorflow2_keras_tests/test_tf2_keras_zeropadding2d.py::TestKerasZeroPadding2D - Failed: tensorflow2_keras_tests/test_tf2_keras_zeropadding2d.py::TestKerasZeroPadding2D::test_keras_zeropadding2d_channels_last: in "parametrize" the number of names (5):
  ['input_names', 'input_shapes', 'input_type', 'padding', 'data_format']
must be equal to the number of values (1):
  ((['x1'], [[1, 3, 8, 7]], tf.float32, ((5, 1), (3, 4))),)
```

Please see the report: https://github.com/openvinotoolkit/openvino/actions/runs/6976462547/job/18988228421?pr=21264